jetstream/external_config.py

```diff
@@ -59,7 +71,18 @@ def from_github_repo(cls) -> "ExternalConfigCollection":
                     )
                 )
 
-        return cls(external_configs)
+            outcomes = []
+
+            for outcome_file in tmp_dir.glob(f"**/{OUTCOMES_DIR}/*/*.toml"):
+                outcomes.append(
+                    ExternalOutcome(
+                        slug=outcome_file.stem,
+                        spec=OutcomeSpec.from_dict(toml.load(outcome_file)),
+                        platform=outcome_file.parent.name,
```

Comment:
As described in https://github.com/mozilla/jetstream-config/pull/42, outcome files are organized in subdirectories that match the app_name, e.g. `fenix/` or `firefox_desktop/`.